### PR TITLE
[MU3] Update templates

### PR DESCRIPTION
**Pull request by njvdberg**
Update of the templates:

1. Enable Vertical Justification Staves.
2. For most templates, enable Indentation on first system.
3. Set appropriate score order for the template. For this several now score orders are added.

An update in <code>file.cpp</code> ensures the new score will use the score order as defined in the template.

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/Documentation/blob/master/CodeGuidelines.md)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

File: share/templates/02-Choral/04-SATB_Closed_Score.mscx [hunk 1]
```diff
@@ -143,46 +160,36 @@ B</longName>
...
             <sigD>4</sigD>
             </TimeSig>
-          <StaffText>
-            <channelSwitch voice="0" name="Soprano"/>
-            <channelSwitch voice="1" name="Alto"/>
-            <channelSwitch voice="2" name="Soprano"/>
-            <channelSwitch voice="3" name="Alto"/>
-            <visible>0</visible>
-            <text>s/a</text>
-            </StaffText>
```
Comment (Jojo-Schmitz) [inline share/templates/02-Choral/04-SATB_Closed_Score.mscx:179]: why removing this?

File: share/templates/02-Choral/04-SATB_Closed_Score.mscx [hunk 2]
```diff
@@ -507,36 +434,20 @@ B</longName>
...
             <sigD>4</sigD>
             </TimeSig>
-          <StaffText>
-            <channelSwitch voice="0" name="Tenor"/>
-            <channelSwitch voice="1" name="Bass"/>
-            <channelSwitch voice="2" name="Tenor"/>
-            <channelSwitch voice="3" name="Bass"/>
-            <visible>0</visible>
-            <text>t/b</text>
-            </StaffText>
```
Comment (Jojo-Schmitz) [inline share/templates/02-Choral/04-SATB_Closed_Score.mscx:533]: why removing this?

File: share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx [hunk 3]
```diff
@@ -98,19 +117,18 @@
         <StaffType group="pitched">
           <name>stdNormal</name>
           </StaffType>
-        <defaultConcertClef>G8vb</defaultConcertClef>
-        <defaultTransposingClef>F</defaultTransposingClef>
+        <defaultClef>F</defaultClef>
```
Comment (Jojo-Schmitz) [inline share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:120]: Why this change?

File: share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx [hunk 4]
```diff
@@ -68,28 +87,28 @@
...
+          </Articulation>
         <Articulation name="marcatoTenuto">
-              <velocity>120</velocity>
-              <gateTime>100</gateTime>
-        </Articulation>
+          <velocity>120</velocity>
+          <gateTime>100</gateTime>
+          </Articulation>
         <Channel name="Soprano">
+          <controller ctrl="32" value="17"/>
```
Comment (Jojo-Schmitz) [inline share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:106]: What is good for? Just curious...

File: share/templates/02-Choral/04-SATB_Closed_Score.mscx (hunk 1 above)
Comment (njvdberg) [reply share/templates/02-Choral/04-SATB_Closed_Score.mscx:179]: They should :-(

File: share/templates/02-Choral/04-SATB_Closed_Score.mscx (hunk 2 above)
Comment (njvdberg) [reply share/templates/02-Choral/04-SATB_Closed_Score.mscx:533]: They should :-(

File: share/templates/02-Choral/04-SATB_Closed_Score.mscx (hunk 1 above)
Comment (Jojo-Schmitz) [reply share/templates/02-Choral/04-SATB_Closed_Score.mscx:179]: So good I checked ;-)

File: share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx (hunk 3 above)
Comment (njvdberg) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:120]: In the old template both voice, female and male, used the same "instrument" <code>voice</code> and adapted the clef for male later. In the new template other instruments are used, <code>Women</code> for S/A and <code>Men</code> for T/B. Now the get automatically the correct clef.

File: share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx (hunk 4 above)
Comment (njvdberg) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:106]: This is the result of saving the templates.

File: share/templates/02-Choral/04-SATB_Closed_Score.mscx (hunk 1 above)
Comment (njvdberg) [reply share/templates/02-Choral/04-SATB_Closed_Score.mscx:179]: Indeed. The choral templates uses invisible staff texts which I initially missed and added them later. Except for one apperently.

File: share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx (hunk 3 above)
Comment (Jojo-Schmitz) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:120]: The nice part of the above was the easy switch between F and G8vb clef for the concert pitch button. Now that is lost

Comment (Jojo-Schmitz) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:120]: Esp. the Tenors tend to hate the F Clef...

File: share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx (hunk 4 above)
Comment (Jojo-Schmitz) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:106]: Hmm, is that then a good reason to keep it?

File: share/templates/02-Choral/04-SATB_Closed_Score.mscx (hunk 1 above)
Comment (Jojo-Schmitz) [reply share/templates/02-Choral/04-SATB_Closed_Score.mscx:179]: These don't really matter, as they are not getting carried over into the score created from the templates though, unfortunately

Comment (Jojo-Schmitz) [reply share/templates/02-Choral/04-SATB_Closed_Score.mscx:179]: They only do show when opening the template itself directly

File: share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx (hunk 3 above)
Comment (njvdberg) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:120]: Not the one I now (me :-))

Comment (Jojo-Schmitz) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:120]: in our choir even one of the basses hates it (being a violinist too)

File: share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx (hunk 4 above)
Comment (njvdberg) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:106]: Is it a reason to remove it? When creating a soprano will save this line. So, why not having it in the template as well?

Comment (Jojo-Schmitz) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:106]: I'd rather first understand what it does

Comment (Jojo-Schmitz) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:106]: I think it switches to another bank in the Soundfont. Hoping it exists (it won't in the default one as far as I can tell?)

Comment (Jojo-Schmitz) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:106]: then again we do something like this for 12 string guitar in instruments.xml, so this might be OK

Comment (Jojo-Schmitz) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:106]: It actually is the bank for the expressive sounds

Comment (Jojo-Schmitz) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:106]: Something that happens automatically anyway, so not needed here, but most probably not harmful either

File: share/templates/02-Choral/04-SATB_Closed_Score.mscx (hunk 1 above)
Comment (njvdberg) [reply share/templates/02-Choral/04-SATB_Closed_Score.mscx:179]: Agree, it was just the reason I initially missed them. Is already corrected, not pushed yet.

Comment (njvdberg) [reply share/templates/02-Choral/04-SATB_Closed_Score.mscx]: Updated.

File: share/templates/02-Choral/04-SATB_Closed_Score.mscx (hunk 2 above)
Comment (njvdberg) [reply share/templates/02-Choral/04-SATB_Closed_Score.mscx]: Updated.

File: share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx (hunk 3 above)
Comment (njvdberg) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx]: Updated. Now all three SATB Closed templates are consistent.

**Review by Jojo-Schmitz — APPROVED** [review]

File: share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx (hunk 4 above)
Comment (njvdberg) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:106]: As a matter of fact, when creating a score using this template in 3.5.2 (or 3.2.3) and save the score, you see these same line popping up in the score.

Comment (Jojo-Schmitz) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx:106]: Yes, as said, happens automagically as ot the SND implementation

File: share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx (hunk 3 above)
Comment (Jojo-Schmitz) [reply share/templates/02-Choral/06-SATB_Closed_Score_+_Piano.mscx]: Thanks!

**Review by vpereverzev — APPROVED** [review]

**Comment by Jojo-Schmitz** [comment]
There's a problem with this PR: "share/templates/Create_New_Score.mscz" is now "share/templates/Create_New_score.mscz", i.e. "Score" became "score", and this causing issues on rebasing

**Comment by njvdberg** [comment]
> There's a problem with this

Solved in #6997
